src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSource.java

```diff
@@ -180,6 +180,10 @@ public BitbucketSCMRepository getBitbucketSCMRepository() {
         return repository;
     }
 
+    public CustomGitSCMSource getGitSCMSource() {
```

Comment:
Not keen on this being public. Maybe we could make it package private or maybe even structure the code differently so we can remove it?